dspace/config/modules/orcid.cfg

```diff
@@ -0,0 +1,130 @@
+
+### Funding mapping ###
+orcid.mapping.funding.title = dc.title
+orcid.mapping.funding.type = 
+orcid.mapping.funding.type.converter = mapConverterDSpaceToOrcidFundingType
+##orcid.mapping.funding.external-ids syntax is <metadatafield>::<type> 
+##The full list of available external identifiers is available here https://pub.orcid.org/v3.0/identifiers
+orcid.mapping.funding.external-ids = oairecerif.internalid::other-id
+orcid.mapping.funding.external-ids = oairecerif.funding.identifier::grant_number
+orcid.mapping.funding.description = dc.description
+orcid.mapping.funding.start-date = oairecerif.funding.startDate
+orcid.mapping.funding.end-date = oairecerif.funding.endDate
+##orcid.mapping.funding.contributors syntax is <metadatafield>::<type>
+orcid.mapping.funding.contributors = project.investigator::lead
+orcid.mapping.funding.organization-relationship-type = isOrgUnitOfProject
+orcid.mapping.funding.amount = oairecerif.amount
+orcid.mapping.funding.amount.currency = oairecerif.amount.currency
+orcid.mapping.funding.amount.currency.converter = mapConverterDSpaceToOrcidAmountCurrency
+
+### Other names mapping ###
+orcid.mapping.other-names = person.name.variant
+orcid.mapping.other-names = person.name.translated
+
+### Keywords mapping ###
+orcid.mapping.keywords = dc.subject
+
+### Country mapping ###
+orcid.mapping.country = person.country
+orcid.mapping.country.converter =
+
+### Person External ids mapping ###
+##orcid.mapping.person-external-ids syntax is <metadatafield>::<type>
+orcid.mapping.person-external-ids = person.identifier.scopus-author-id::SCOPUS
+orcid.mapping.person-external-ids = person.identifier.rid::RID
+
+### Researcher urls mapping ###
+orcid.mapping.researcher-urls = oairecerif.identifier.url
+
+### Organization mapping ###
+orcid.mapping.organization.title = organization.legalName
+orcid.mapping.organization.country = organization.address.addressCountry
+orcid.mapping.organization.city = organization.address.addressLocality
+
+##orcid.mapping.organization.identifiers syntax is <metadatafield>::<source>
+##Valid organization identifier source values are ${orcid.validation.organization.identifier-sources}
+orcid.mapping.organization.identifiers = organization.identifier.crossrefid::FUNDREF
+orcid.mapping.organization.identifiers = organization.identifier.rin::RINGGOLD
+orcid.mapping.organization.identifiers = organization.identifier.lei::LEI
+
+### Contributor mapping ###
+orcid.mapping.contributor.email = person.email
+orcid.mapping.contributor.orcid = person.identifier.orcid
+
+#------------------------------------------------------------------#
+#--------------ORCID OBJECTS VALIDATION CONFIGURATIONS-------------#
+#------------------------------------------------------------------#
+
+orcid.validation.affiliation.enabled = true
+orcid.validation.work.enabled = true
+orcid.validation.funding.enabled = true
+
+orcid.validation.organization.identifier-sources = RINGGOLD
```

Comment:
Please add comments to these validation settings (both above and below).  For example, I don't know what these "identifier-sources" are and where they come from.